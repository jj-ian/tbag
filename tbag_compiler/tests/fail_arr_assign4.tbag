/* Authors: Maria */
true {
        testvar();
        endgame;
}

func int testvar() {
        int i;
        int j = i [0];
        return 0;
}
