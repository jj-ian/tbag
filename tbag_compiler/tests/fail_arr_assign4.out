Fatal error: exception Failure("Array access must be used on an array
type")
